src/components/colliders/BaseCollider.vue

```diff
@@ -86,5 +98,7 @@ onUnmounted(() => {
 </script>
 
 <template>
-  <slot></slot>
+  <TresGroup>
```

Comment:
The reason to use a `<TresGroup>` here is what @JaimeTorrealba mentioned here https://github.com/Tresjs/rapier/pull/141#issuecomment-2437283852 ?

Reply:
Yes, exactly!

Reply:
I was thinking about this, a `<TresGroup />` makes no sense here... Maybe add a "dummy" object like a "TresObject3D".

Something that can emit an event (a Tres-vue component) but also something that makes sense when you inspect the hierarchy tree in Three.js.

Reply:
Yes @JaimeTorrealba, I used a `Group` to match what was done in the `RigidBody` component,

But in reality, both of them should use `Object3D` instead.

Reply:
I see, I mean, we use TresGroup a lot on cientos for some of the abstractions, so I'm fine either way. 

@JaimeTorrealba what does the dummy object would look like?

Reply:
My suggestion is exactly that a "TresObject3D" (we generally used a dummy object3D in instanceMesh) it would solve the issue, also we can add (optionally) some `:name="ColliderId"` attribute, this way if you look into the threeJs scene makes more sense than a plain TresGroup, right? Or any other data/value that identify as a collider. All these are just suggestions., @alvarosabu.

@Neosoulink IDK if I misunderstand your last comment, but no `<RigidBody>` receive slots, it makes sense to be a `TresGroup`

Reply:
No, there's no misunderstanding,
I was just clarifying the topic of using an `Object3D`.